### PR TITLE
Newsletter: New chapters in the OS dev tutorials

### PR DESCRIPTION
**Pull request by andre-richter**

### CODE REVIEW

File: content/2019-11-28-newsletter-21.md [hunk 1]
```diff
@@ -43,6 +43,10 @@ If you want to mention something in [the next newsletter], send us a pull reques
 
 - [@japaric] released version v0.1.0 of [`ufmt`], a smaller (6-40x) and faster (2-9x) alternative to `core::fmt` that's free of dynamic dispatch, recursion (where possible) and panicking branches. `ufmt` is *not* a drop-in replacement for `core::fmt` but it provides `uDebug` and `uDisplay` traits and a family of `uwrite!` macros. In this release the `uwrite!` macros, and the rest of the crate, work on the stable channel.
 
+- [@andre-richter] added two new tutorials for the `Operating System development tutorials in Rust on the Raspberry Pi` series:
+    - Tutorial 11: [Virtual Memory](https://github.com/rust-embedded/rust-raspi3-OS-tutorials/tree/master/11_virtual_memory)
+    - Tutorial 12: [CPU Exceptions: Part 1](https://github.com/rust-embedded/rust-raspi3-OS-tutorials/tree/master/12_exceptions_part1)
```
Comment (adamgreig) [inline content/2019-11-28-newsletter-21.md]: This link 404s

Comment (andre-richter) [reply content/2019-11-28-newsletter-21.md]: Ooh, thanks for catching this!

**Comment by andre-richter** [comment]
> Would you mind squashing your commits?

Done

**Pull request merged**